mappings/net/minecraft/server/world/ServerTickScheduler.mapping

```diff
@@ -1,9 +1,12 @@
 CLASS bhy net/minecraft/server/world/ServerTickScheduler
 	FIELD a invalidObjPredicate Ljava/util/function/Predicate;
 	FIELD b idToName Ljava/util/function/Function;
```

Comment:
`deserializer`. There is no "name", it is Identifier->T used for deserialization.